### ISSUE TITLE
 Error not receiving a parameter

### ISSUE THREAD

**Issue by devkyo** [BABAERON/PIP#51]
Hi, I'm using the framework, I like it because it's simple.
I have a problem:
What happens if I don't receive a parameter in a function? I selling error:

`Fatal error: Uncaught ArgumentCountError: Too few arguments to function Usuario::restablecer(), 0 passed in C:\laragon\www\lidcon\system\pip.php on line 176 and exactly 1 expected `

I tried to validate if the variable is defined but still I have the same error

**Comment by karansapkale** [comment]
Hi, you must have forgotten to pass the parameter required for the function.

Can you share piece of code to get better understanding.

**Comment by devkyo** [comment]
Hello,

I meant what happens if I have a function that receives a parameter, and I don't send it a parameter?
What would you show me?
blank page or an error?

In my case it shows an error.

**Comment by devkyo** [comment]
check my code http://prntscr.com/pte99x

**Comment by karansapkale** [comment]
Simple do this:

function restablecer($hash="")
{

**Comment by devkyo** [comment]
Yes, solve it by giving it a default value and then validating with if
